app/code/Magento/Catalog/Block/Product/ListProduct.php

```diff
@@ -68,12 +69,18 @@ class ListProduct extends AbstractProduct implements IdentityInterface
      */
     protected $categoryRepository;
 
+    /**
+     * @var CollectionLoader
+     */
+    protected $collectionLoader;
```

Comment:
New class parameter should be private as we are trying to avoid using protected.